packages/@jsii/go-runtime/jsii-runtime-go/helpers.go

```diff
@@ -2,23 +2,45 @@ package jsii
 
 import "time"
 
+type Type interface {
```

Comment:
Is there a reason why this is exported, but `numberType` is not?

Reply:
I don't believe the `Type` constraint interface has to be exported at all. It can still apply without it being a public type exported by the package.

I think it's a design decision of whether it should be visible to users or not. I'm thinking that it might be useful to export it, but I can't think of a use case for exporting it.

What do you think? Export or no?

Reply:
I've removed them, since they're not required to be public, on the basis that it's possible to make things public later, but not to remove them.

Reply:
Yeap, I'd start off with them hidden, and we can expose them later if necessary... The other way around is not as easy ^^